src/main/java/org/reaktivity/nukleus/http2/internal/Http2WriteScheduler.java

```diff
@@ -406,6 +416,14 @@ private void flush()
         }
         writer.flush();
 
+        for(Http2Stream stream : connection.http2Streams.values())
```

Comment:
Does this produce garbage via `Iterator`?
If so, perhaps use `forEach` instead, allowing a more efficient internal implementation of `forEach` than the default which also uses `Iterator`.

Reply:
Int2ObjectHashMap returns the same ValueIterator. So I left the code as it is.